torchtitan/models/llama/model.py

```diff
@@ -421,7 +420,7 @@ def forward(self, tokens: torch.Tensor):
             torch.Tensor: Output logits after applying the Transformer model.
 
         """
-        _bsz, seqlen = tokens.shape
```

Comment:
similarly, `_bsz` is unused, so just remove

Reply:
if it helps readability to know the shape, I can keep it and maybe rename it to `_bs`?

Reply:
Although not used, it improves code readability -- it tells how many dimensions `tokens` has, and what they are. So IMO I'd wish they are kept. Also, the "unusedness" has been indicated using the `_` prefix.

Reply:
> if it helps readability to know the `tokens.shape` is (batch size, sequence length), I can keep it and maybe rename it to `_bs`?

just saw this message, yeah I agree

Reply:
changed it to `_bs`